src/platform/plugins/shared/metrics_experience/server/lib/fields/retrieve_fieldcaps.ts

```diff
@@ -24,42 +24,38 @@ export async function retrieveFieldCaps({
   fields?: Fields;
   timerange: EpochTimeRange;
 }) {
-  const dataStreamFieldCapsMap = new Map<
+  const indexFieldCapsMap = new Map<
     string,
     Record<string, Record<string, FieldCapsFieldCapability>>
   >();
 
-  // First, resolve the index pattern to get data streams
+  // First, resolve the index pattern to get data streams or indices
   const resolveResponse = await esClient.indices.resolveIndex({
     name: indexPattern,
     expand_wildcards: 'open',
   });
```

Comment:
We need to update the elsaticsearch package to be able to pass `mode: 'times_series'` parameter.

Reply:
https://github.com/elastic/kibana/pull/242182

Reply:
There is a bug in the API to retrieve remote clusters `data_streams` if `mode` is passed with `time_series` https://github.com/elastic/elasticsearch/issues/137814. So we couldn't do it now.